src/components/modal/modal.js

```diff
@@ -28,7 +28,7 @@ export const Modal = ({ setModal, video }) => {
           headers: header,
         }
       );
-      setPlaylistName(response.data.playlists);
+      setPlaylist(response.data.playlists);
       setCreatePlaylist(false);
     } catch (e) {
       console.log(e.message);
```

Comment:
One Suggestion instead of consoling this, you can show the error message in view (toast messages)